psse/psse-model/src/main/java/com/powsybl/psse/model/pf/PsseValidation.java

```diff
@@ -0,0 +1,513 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model.pf;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+import java.util.stream.Collectors;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.powsybl.psse.model.PsseVersion;
+import static com.powsybl.psse.model.PsseVersion.Major.V35;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+public class PsseValidation {
+
+    public PsseValidation(PssePowerFlowModel model, PsseVersion psseVersion) {
+        Objects.requireNonNull(model);
+        warnings = new ArrayList<>();
+        validCase = true;
+
+        validate(model, psseVersion);
+        writeWarnings();
+    }
+
+    public List<String> getWarnings() {
+        return warnings;
+    }
+
+    public boolean isValidCase() {
+        return validCase;
+    }
+
+    private void writeWarnings() {
+        LOGGER.warn("PSS/E Validation ...");
```

Comment:
only emit "PSS/E validation ..." as a warning if the list of warnings is not empty (no warnings in the output if everything is ok)

Reply:
Done